src/pgen/homologous.cpp

```diff
@@ -0,0 +1,137 @@
+// © 2021. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract
+// 89233218CNA000001 for Los Alamos National Laboratory (LANL), which
+// is operated by Triad National Security, LLC for the U.S.
+// Department of Energy/National Nuclear Security Administration. All
+// rights in the program are reserved by Triad National Security, LLC,
+// and the U.S. Department of Energy/National Nuclear Security
+// Administration. The Government is granted for itself and others
+// acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works,
+// distribute copies to the public, perform publicly and display
+// publicly, and to permit others to do so.
+
+#include <cmath>
+#include "geometry/geometry.hpp"
+#include "monopole_gr/monopole_gr.hpp"
+#include "pgen/pgen.hpp"
+#include "/Work/Phoebus/phoebus/Tests/GRsolver/reader.hpp"
```

Comment:
you don't have enough cmake for the linker to link correctly yet because the cmake is all in yoru `Tests` directory. You need to hook into the rest of teh phoebus build system

Reply:
Okay so I changed this to Tests/Reader/reader.hpp. I added CMakelists.txt in the Tests directory which has add_subdirectory of Reader in it. I also added corresponding link in phoebus's CMakelist.txt.

Reply:
:+1: does it compile now?

Reply:
No it says Monopole GR is only enabled with TOV.